cclib/scripts/ccframe.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Script for writing data tables from computational chemistry files."""
+
+import sys
+import os.path
+import argparse
+
+try:
+    import pandas as pd
+    _has_pandas = True
+except ImportError:
+    _has_pandas = False
```

Comment:
`_has_pandas` doesn't appear to be used anywhere in this script.

Reply:
2a2e4e6 made use of it for failing cleanly if pandas can't be imported.